### PR TITLE
HTML and CSS Updated

### PR DESCRIPTION
**Pull request by rmauritsson**

### CODE REVIEW

**Comment by samgaco** [comment]
### Hi @rmauritsson :vulcan_salute: 

### Great job so far! :+1:  The site looks nice :100: There are some things to improve before proceeding though:

* The elements within the **navigation bar** should be `<button>` tags, and they should be positioned with either flex or floats.

* The sentences  _What will your verse be?_ and _Everyone has something to share..._ should be images set as backgrounds of a` <div> `tag, and at the same tame enclosed in `<a>` tags. _(now you have them as embedded images)_

* The sentence _"Watch the "Your Verse" film and explore the stories"_ should be a text with a hover underline effect.  **Suggestion**: The play could be an icon from [fontsawesome](https://fontawesome.com/).

* You should **add a readme file**, with a project description and is suggested to have your contacts and a live demo of the site.

* The **horizontal scroll bar should not be showing**.


Recommendations:

*  Is recommended to have the links in the footer arranged with either floats or display flex.

### 
That's it. Please :pray: commit these changes and submit back for a code review when you are done. Happy coding and regards! :muscle:

**Comment by rmauritsson** [comment]
Updated as per the recommendations

**Comment by Ispirett** [comment]
#Reviewing

File: index.html [hunk 1]
```diff
@@ -1,32 +1,72 @@
...
+                </ul>
+            </nav>
+            <div class="one"></div>
+            <div class="two"></div>
+            <div class="three">
+                <div class="three-1"></div>
+                <div class="three-2"><i class="far fa-play-circle"></i></div>
+            </i></div>
+    </main>
+    <footer>
```
[the remark this answers is not in the document]
Comment (rmauritsson) [reply index.html]: Display flex changed the entire layout so i used inline-flex.
Is that not acceptable?

**Comment by Ispirett** [comment]
## Reviewing

**Comment by Ispirett** [comment]
Good job 
:+1:  with the changes you can **Complete** this project :+1: 
- [ ] **Optional** The background image should be inside `<div>` instead of <a> right now the div is just the empty.

**Pull request merged**